sass/components/_timepicker.scss

```diff
@@ -39,12 +44,20 @@
 	cursor: pointer;
 }
 
-.timepicker-span-hours {
-  margin-right: 3px;
+.timepicker-input-hours {
+  height: 4rem !important;
+  width: 30% !important;
+  border-bottom: 0px !important;
+  font-size: 4rem !important;
+  margin-right: 3px !important;
 }
 
-.timepicker-span-minutes {
-  margin-left: 3px;
+input.timepicker-input-minutes {  //change class name
```

Comment:
Did you forget to remove the `// change class name` comment ?

Reply:
Sorry for leaving unnecessary comment. It will be removed.